src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebar.tsx

```diff
@@ -90,15 +94,30 @@ export const ArtworkSidebar: React.FC<
 
   const artworkEcommerceAvailable = !!(isAcquireable || isOfferable)
 
+  // TODO: Arta Estimate widget experiment! Either remove or properly instrument
   const artsyShippingEstimateEnabled = useFeatureFlag(
     "emerald_shipping-estimate-widget",
   )
   const allArtsyShipping =
     !!artsyShippingDomestic && !!artsyShippingInternational
   const artsyImpliedShipping =
     !!artsyShippingDomestic && !internationalShippingFee
+  const isOneEdition = isEdition && editionSets && editionSets.length === 1
+  let isWeightArtaEstimatable = true
+  // TODO: for now ignoring weight for works with one edition set.
+  // Need to either support in MP or roll data to the artwork level.
+  // TODO: have standard shippingWeightKg in MP?
+  if (!isEdition && shippingWeight && shippingWeightMetric) {
+    isWeightArtaEstimatable =
+      shippingWeightMetric === "kg"
+        ? shippingWeight <= 68
+        : shippingWeight <= 150
```

Comment:
We talked about this outside of github - I suggested putting this instead in the function that creates the Shipping estimate request body (which may already return null based on the artwork shape and short-circuit the estimate) would be closer to where this kind of validation needs to happen. @oxaudo pointed out that at that point we are loading an entire extra library via script and I agreed it's fine to leave here for now.

minor: Could we update the comment to describe what this check is doing and make tickets or raise the TODOs in slack to make sure we come back to it

Reply:
I'll add a placeholder card to 'make a decision and either clean or ticket the work once the future of Arta estimate experiment is decided". Does that sound right?